crates/storage/provider/src/db_provider/block.rs

```diff
@@ -16,3 +16,30 @@ impl<DB: Database> HeaderProvider for ProviderImpl<DB> {
         }
     }
 }
+
+impl<DB: Database> BlockProvider for ProviderImpl<DB> {
+    fn chain_info(&self) -> Result<ChainInfo> {
+        Ok(ChainInfo {
+            best_hash: Default::default(),
+            best_number: 0,
+            last_finalized: None,
+            safe_finalized: None,
+        })
```

Comment:
TODO: This should be handled by Engine API channel after we're synced?

Does it make sense to have this? I think we may use this in the RPC.

Reply:
This should report the latest synced block, so the result of the last stage in the pipeline (doesn't exist yet). Will probably not be used by the networking component because it would require a lookup in the database on every handshake, but should be used by the RPC I think